dev/pip-sanity-check.py

```diff
@@ -18,6 +18,8 @@
 from __future__ import print_function
 
 from pyspark.sql import SparkSession
+from pyspark.ml.param import Params
+from pyspark.mllib.linalg import *
```

Comment:
This just checks one sub component from each, we could import each with rename I suppose but not sure it would do much?